app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,34 @@
+class SessionsController < ApplicationController
+
+  def new
+
+  end
+
+  def create
+    user = User.find_by_email(params[:email])
+    # If the user exists AND the password entered is correct.
+    if user && user.authenticate(params[:password])
+      # Save the user id inside the browser cookie. This is how we keep the user 
+      # logged in when they navigate around our website.
+      session[:user_id] = user.id
+      redirect_to '/index'
+    else
+    # If user's login doesn't work, send them back to the login form.
+      flash[:error] = 'Incorrect Username or password!'
+      redirect_to '/login'
+    end
+  end
+
+
+
+  def destroy
+    session[:user_id] = nil
+    redirect_to '/login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.